yoti-sdk-api/src/main/java/com/yoti/api/client/docs/session/create/SessionSpec.java

```diff
@@ -146,19 +167,63 @@ public Boolean getBlockBiometricConsent() {
         return blockBiometricConsent;
     }
 
+    /**
+     * The options that define if a session will be required to be performed
+     * using In-Branch Verification
+     *
+     * @return the IBV options
+     */
+    public IbvOptions getIbvOptions() {
+        return ibvOptions;
+    }
+
+    /**
+     * The deadline that the session needs to be completed by.
+     *
+     * @return the session deadline
+     */
+    public ZonedDateTime getSessionDeadline() {
+        return sessionDeadline;
+    }
+
+    /**
+     * Defines a required identity profile within the scope of a trust framework and scheme.
+     *
+     * @return Identity Profile
+     */
+    public Object getIdentityProfile() {
```

Comment:
I don't know if returning `Object` here is really a good idea.  As we spoke about previously, if we want to add a POJO in the future that references this, we cannot add an overloaded method called `getIdentityProfile` that returns that POJO (as `Object` and the new class will conflict).  Do you think we should return `Map<String, Object>` here?

Reply:
I initially started with `Map<String, Object>` to change it, during review, in `Object` as it was done in .NET

Once the final structure is defined we should use a strict Typed class hierarchy to help integrators to build correctly the requests

Reply:
This will cause it to be a breaking change though.  We can't change the method signature from returning an `Object` to a strictly typed object that we have defined in the future.

What we could do however is have the method return `Map<String, Object>`, and then deprecate this method later, without it being a breaking change to the current API.

Reply:
Worth to mention is that Object allows integrators to use both Map and Jackson annotated classes. Rolling back to Map for now.